concepticondata/conceptlists/Sawka-2019-201.tsv

```diff
@@ -0,0 +1,202 @@
+Sawka-2019-201-51	51	bee	animal	665	BEE
+Sawka-2019-201-52	52	bird	animal	937	BIRD
+Sawka-2019-201-53	53	buffalo	animal	934	BUFFALO
+Sawka-2019-201-54	54	cat	animal	1208	CAT
+Sawka-2019-201-55	55	crocodile	animal	1857	CROCODILE
+Sawka-2019-201-56	56	dog	animal	2009	DOG
+Sawka-2019-201-57	57	elephant	animal	1290	ELEPHANT
+Sawka-2019-201-58	58	fish	animal	227	FISH
+Sawka-2019-201-59	59	goat	animal	1502	GOAT
+Sawka-2019-201-60	60	head louse	animal	310	HEAD LOUSE
+Sawka-2019-201-61	61	hippo	animal	3145	HIPPOPOTAMUS
+Sawka-2019-201-62	62	lion	animal	1386	LION
+Sawka-2019-201-63	63	rhino	animal	2591	RHINOCEROS
+Sawka-2019-201-64	64	snake	animal	730	SNAKE
+Sawka-2019-201-65	65	spider	animal	843	SPIDER
+Sawka-2019-201-66	66	wild animal	animal	958	WILD ANIMAL
+Sawka-2019-201-67	67	bull	animal	1008	BULL
+Sawka-2019-201-68	68	calf	animal	1989	CALF
+Sawka-2019-201-69	69	cow	animal	1007	COW
+Sawka-2019-201-70	70	chicken	animal	1318	CHICKEN
+Sawka-2019-201-71	71	cock	animal	1511	ROOSTER
+Sawka-2019-201-72	72	arrow	thing	977	ARROW
+Sawka-2019-201-73	73	ash	thing	646	ASH
+Sawka-2019-201-74	74	axe	thing	677	AXE
+Sawka-2019-201-75	75	bark (of tree)	thing	1204	BARK
+Sawka-2019-201-76	76	burden	thing	1679	LOAD
+Sawka-2019-201-77	77	chair	thing	1476	CHAIR
+Sawka-2019-201-78	78	cloud	thing	1489	CLOUD
+Sawka-2019-201-79	79	cooking pot	thing	3653	COOKING POT
+Sawka-2019-201-80	80	darkness	thing	11	DARKNESS
+Sawka-2019-201-81	81	daytime	thing	1225	DAY (NOT NIGHT)
+Sawka-2019-201-82	82	dew	thing	1977	DEW
+Sawka-2019-201-83	83	dust	thing	2	DUST
+Sawka-2019-201-84	84	earth	thing		
+Sawka-2019-201-85	85	egg	thing	744	EGG
+Sawka-2019-201-86	86	feather	thing	1201	FEATHER
+Sawka-2019-201-87	87	field	thing	212	FIELD
+Sawka-2019-201-88	88	finger millet	thing	3265	FINGER MILLET
+Sawka-2019-201-89	89	fire	thing	221	FIRE
+Sawka-2019-201-90	90	firewood	thing	10	FIREWOOD
+Sawka-2019-201-91	91	flower	thing	239	FLOWER
+Sawka-2019-201-92	92	grass	thing	606	GRASS
+Sawka-2019-201-93	93	hardship	thing		
+Sawka-2019-201-94	94	hoe	thing	284	HOE
+Sawka-2019-201-95	95	hole	thing	1667	HOLE
+Sawka-2019-201-96	96	honey	thing	942	HONEY
+Sawka-2019-201-97	97	house	thing	1252	HOUSE
+Sawka-2019-201-98	98	hunger	thing	1864	HUNGER
+Sawka-2019-201-99	99	iron	thing	621	IRON
+Sawka-2019-201-100	100	knife	thing	1352	KNIFE
+Sawka-2019-201-101	101	knot	thing	760	KNOT
+Sawka-2019-201-102	102	lake	thing	624	LAKE
+Sawka-2019-201-103	103	light	thing	630	LIGHT (RADIATION)
+Sawka-2019-201-104	104	meat	thing	634	MEAT
+Sawka-2019-201-105	105	medicine	thing	1372	MEDICINE
+Sawka-2019-201-106	106	milk	thing	635	MILK
+Sawka-2019-201-107	107	month	thing	1370	MONTH
+Sawka-2019-201-108	108	moon	thing	1313	MOON
+Sawka-2019-201-109	109	mountain	thing	639	MOUNTAIN
+Sawka-2019-201-110	110	mourning - sadness	thing	3673	MOURNING
+Sawka-2019-201-111	111	name	thing	1405	NAME
+Sawka-2019-201-112	112	night	thing	1233	NIGHT
+Sawka-2019-201-113	113	path	thing	2252	PATH
+Sawka-2019-201-114	114	porridge	thing		
```

Comment:
Might be 3651, but need to be verified against source.

Reply:
The source does not specify anything, should I still map it to 3651?